challenge.js

```diff
@@ -56,6 +56,6 @@
  *	Paste the URL of the MDN page below.
  */
 
-// ANSWER
+// ANSWER https://developer.mozilla.org/en-US/docs/Web/API/Node/appendChild
```

Comment:
Great. Since the method you mentioned in your answer to the first question sounds kind of like `document.createElement`, here is a snippet using that and this method to create a new HTML element and stick it on the DOM at a location of our choosing:

```js
var div = document.querySelectorAll("body > div")[0]; // the zeroth element of the set of all divs that are immediate children of the body
var e = document.createElement('textarea'); // this textarea element is not on the DOM yet
div.appendChild(e); // mission accomplished
```

1